RInAppMessaging/Classes/Extensions/Int32+IAM.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+extension Int32 {
+    /// Exponential backoff
+    mutating func increaseBackOff() {
+        self = multipliedReportingOverflow(by: 2).partialValue
+    }
+
+    /// Exponential backoff with a random value
+    mutating func increaseRandomedBackoff() {
```

Comment:
Randomed -> Randomized is better I think

Reply:
Ok fixed.